src/main/java/alfio/manager/TicketReservationManager.java

```diff
@@ -796,7 +800,7 @@ private Locale findReservationLanguage(String reservationId) {
 
     public void deleteOfflinePayment(Event event, String reservationId, boolean expired, boolean credit, boolean notify, String username) {
         TicketReservation reservation = findById(reservationId).orElseThrow(IllegalArgumentException::new);
-        Validate.isTrue(reservation.getStatus() == OFFLINE_PAYMENT || reservation.getStatus() == DEFERRED_OFFLINE_PAYMENT, "Invalid reservation status");
+        Validate.isTrue(List.of(OFFLINE_PAYMENT, DEFERRED_OFFLINE_PAYMENT, CUSTOM_OFFLINE_PAYMENT).contains(reservation.getStatus()), "Invalid reservation status");
```

Comment:
this code might be rewritten as:
`Validate.isTrue(EnumSet.of(OFFLINE_PAYMENT, DEFERRED_OFFLINE_PAYMENT, CUSTOM_OFFLINE_PAYMENT).contains(reservation.getStatus()), "Invalid reservation status");`